packages/origin/src/blockchain-facade/Certificate.ts

```diff
@@ -388,3 +375,17 @@ export class Entity extends TradableEntity.Entity implements ICertificate {
         return returnEvents;
     }
 }
+
+export async function claimCertificates(
```

Comment:
Would be cool if we would also add `await certificate.claim()` method to Certificate.ts. This would allow claiming a certificate without passing in a an array of IDs.